### PR TITLE
#28357 -- Prepopulated_fields doesn't work for admin.StackedInline

### PR DESCRIPTION
**Pull request by jkhlr**
This PR fixes #28357, which was caused by a broken jQuery selector.

### CODE REVIEW

**Comment by felixxm** [comment]
Test is required.

**Comment by jkhlr** [comment]
I added a test for the scenario defined in the issue description ✅

**Comment by felixxm** [comment]
buildbot, test on selenium.

File: django/contrib/admin/static/admin/js/prepopulate_init.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
     'use strict';
     var fields = $('#django-admin-prepopulated-fields-constants').data('prepopulatedFields');
     $.each(fields, function(index, field) {
-        $('.empty-form .form-row .field-' + field.name + ', .empty-form.form-row .field-' + field.name).addClass('prepopulated_field');
```
[the remark this answers is not in the document]
Comment (felixxm) [reply django/contrib/admin/static/admin/js/prepopulate_init.js]: It turned out (after detailed investigation :male_detective:) that only `StackInline`'s with lines in `fieldset` that have only a single field are affected, e.g.
```html
<fieldset class="module aligned">
    <div class="form-row field-slug">
```
The current selector works fine with multiple fields in a line, e.g.
```html
<fieldset class="module aligned">
    <div class="form-row field-title field-slug">
        <div class="fieldBox field-title">
    ...
</fieldset>
```
That's why current tests works (`tests.admin_views.testsSeleniumTests.test_prepopulated_fields`).

**Comment by felixxm** [comment]
@jkhlr I added selenium tests instead of JS tests. Please remember to compress JavaScript (see [documentation](https://docs.djangoproject.com/en/3.0/internals/contributing/writing-code/javascript/#compressing-javascript)):
```
python django/contrib/admin/bin/compress.py
```
and check failing tests and my comments.

**Comment by felixxm** [comment]
buildbot, test on selenium.

File: django/contrib/admin/static/admin/js/prepopulate_init.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jkhlr) [reply django/contrib/admin/static/admin/js/prepopulate_init.js]: Agreed to do the minimal patch. I implemented your suggestions, tests are running through now.

**Comment by felixxm** [comment]
buildbot, test on selenium.

File: django/contrib/admin/static/admin/js/prepopulate_init.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (felixxm) [reply django/contrib/admin/static/admin/js/prepopulate_init.js]: @jkhlr You missed my second comment. We need all three options:
 
- `'.empty-form .form-row.field-' + field.name` (`StackedInline` with a single field in a line)
- `'.empty-form .form-row .field-' + field.name` (`StackedInline` with multiple fields in a line)
- `'empty-form.form-row .field-' + field.name` (`TabularInline`)

**Comment by felixxm** [comment]
Please don't force-push your changes, it will be easier to review them.

**Comment by jkhlr** [comment]
buildbot, test on selenium.

**Comment by felixxm** [comment]
Closing due to inactivity.

**Comment by FlorianWendelborn** [comment]
@felixxm what kind of activity did you expect to occur here?

**Comment by felixxm** [comment]
Selenium tests don't work, so that would be a good start.

**Pull request closed without merging**